modules/role/manifests/st2.pp

```diff
@@ -22,4 +20,12 @@
       include ::profile::hubot
     }
   }
+
+  if $_enterprise_token {
+    include ::profile::st2flow
+    include ::profile::st2rbac
+    include ::profile::enterprise_auth_backend_ldap
+  } else {
+    include ::profile::auth_backend_pam
```

Comment:
Then the right answer is to fence this somehow after we add elements to the installer.

@dzimine are we "releasing" enterprise and community, or just community this week? We'll have to solve this concern if both